master/internal/experiment_utils.go

```diff
@@ -57,50 +56,3 @@ func checkpointFromCheckpointMetrics(metrics workload.CheckpointMetrics) model.C
 		Format:    metrics.Format,
 	}
 }
-
-func saveWorkload(db *db.PgDB, w workload.Workload) error {
-	switch w.Kind {
```

Comment:
Jeez.. deleting these makes me so happy, just because I've seen these error messages so much while debugging issues. I'll try to think about the class of bugs they surfaced though and make sure we're covered. But I think it's mostly covered by the new messages that complain about inserting/updating duplicate/existing training and validation workloads, and those getting bubbled up from http calls now.